packages/ffe-account-selector-react/src/account-selector-multi/AccountSuggestionMulti.tsx

```diff
@@ -15,7 +15,7 @@ export function AccountSuggestionMulti({
     selected = false,
 }: AccountSuggestionMultiProps) {
     const { accountNumber, balance, name, currencyCode } = account;
-    const hasBalance = balance !== null;
+    const hasBalance = balance !== undefined;
```

Comment:
Tusen takk før att du fikser. Det du føreslår elelr slik Du får velge :) 
```suggestion
    const hasBalance =  typeof balance === 'number'
```

Reply:
Edit ser nu att det også kan vare en string....

Reply:
Balance kan også være string i følge typen i `Account`, så tenker `undefined` sjekk passer best da